src/main/java/MyMap.java

```diff
@@ -0,0 +1,89 @@
+import java.util.ArrayList;
+
+/**
+ * All you need to do for this microlab is take the Table and Entry from the last one and make Entry a nested class.
+ * Think about how nested classes should work with generics.
+ */
+public class MyMap<K, V> {
+
+    private ArrayList<Entry> entries;
```

Comment:
umm, shouldn't this be myarraylist?

Reply:
Oh there was a part3 challenge part to create a Map. I have a MyArrayList as well

Reply:
Oh ! sorry I misread. Yeah it should be. I forgot to use the arraylist that I made for that one